### ISSUE TITLE
b2b users can see embedded report

### ISSUE THREAD

**Issue by Ellestad1995** [MicrosoftDocs/powerbi-docs#2019]
> Azure B2B Guest users accounts are not supported. Users see the Power BI logo that shows the part is loading, but it doesn't show the report.

Under known issues and limitations it says b2b users cant see the embedded report. In my testing this works fine. The documentation should be updated to reflect til feature.

b2b users was invited like other org users. They could see the report that was embedded in Sharepoint like other org users.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0fc5abbd-e082-d1e1-109d-6f6cec8854a1
* Version Independent ID: f8affce3-98f2-0cc6-b4de-d9af7ced0fbe
* Content: [Embed with report web part in SharePoint Online - Power BI](https://docs.microsoft.com/en-us/power-bi/service-embed-report-spo#feedback)
* Content Source: [powerbi-docs/service-embed-report-spo.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/live/powerbi-docs/service-embed-report-spo.md)
* Service: **powerbi**
* Sub-service: **powerbi-service**
* GitHub Login: @KesemSharabi
* Microsoft Alias: **kesharab**

**Comment by KesemSharabi** [comment]
Hi @Ellestad1995,

If the user can access your SharePoint, he will be able to see the report too. Is that the scenario you're describing?

**Comment by Ellestad1995** [comment]
Yes. Invited b2b users can see the report, which the documentation says they cannot.

**Comment by KesemSharabi** [comment]
Hi @Ellestad1995,

The documentation states that **guest** user accounts are not supported (this is only one of the clauses for troubleshooting an error).

The behavior you're describing seems to be aligned with step 8 in [Grant access to reports](https://docs.microsoft.com/power-bi/collaborate-share/service-embed-report-spo#grant-access-to-reports).

Am I missing something in your scenario?

Thanks,

Kesem

**Comment by Ellestad1995** [comment]
Hi @KesemSharabi 

Maybe I misinterpreted the documentation?

> guest user accounts are not supported

I invited guest users to the PowerBI report. The PowerBI report is embedded in Sharepoint. 
Per the documentation I would think that the guest users would NOT be able to see the PowerBI report. BUT the guest users do see the report in the Sharepoint site.

There is no error, just that guest users are able to see the report, even though the documentation says: "guest user accounts are not supported".
